spec/integration/admin_api/apis_routes_spec.lua

```diff
@@ -109,6 +110,7 @@ describe("Admin API", function()
         assert.truthy(body_page_1.data)
         assert.equal(3, table.getn(body_page_1.data))
         assert.truthy(body_page_1.next)
+        assert.equal(10, body_page_1.total)
```

Comment:
I wouldn't add that everywhere because it makes changing the tests very painful now (for all entities). Also semantically this test (and the others) is not explaining it is testing the total count. Instead, a `describe("total count")` with several additions and deletions and specifically testing the total count is better.